### PR TITLE
Remove Inconsistent KSI and changes views from the DB

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18423

I wasn't sure if i should add these drop commands to the post migration scripts folder or if i should do all the other fancy stuff and add them to sql templates and create down/up migrations? @johnclary lmk what you think makes the most sense

## Testing
**URL to test:** 
Local data model

**Steps to test:**
1. Run the commands to drop the views, thats it!

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

**Review by mddilley — COMMENTED** [review]
I ran these commands, and they work but the Hasura metadata also needs cleanup or else it ends up being inconsistent with the DB schema. That might be part of a different task, but I wanted to check!

File: atd-toolbox/data_model/python/sql_templates/recommendations_and_notes_up.sql
Comment (johnclary) [inline atd-toolbox/data_model/python/sql_templates/recommendations_and_notes_up.sql:1]: we now have to sets of migrations to split out from this file where i have crammed them. i will do that in a follow up very soon

**Review by johnclary — APPROVED** [review]
@mddilley thanks for flagging the metadata—i fixed that up ✅

**Review by mddilley — APPROVED** [review]
Spun this up locally, and I no longer see the two views or the metadata associated with them. Thanks! 🚢 🚀

**Pull request merged**